packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -47,7 +47,7 @@ describe('Sequence - ', () => {
       }
     }
     // bind user defined sequence
-    app.bind('sequence').toClass(MySequence);
+    app.sequence(MySequence);
```

Comment:
My vote is currently for Interface so we don't force the users to extend anything. Nice thing about abstract class is we could provide default implementations (like run) and allow the users to simply override the base impls. Pro/cons depending on the direction/UX we want.